pkg/commands/process/settings/rules/ruby/third_parties/segment/.snapshots/TestRubyThirdPartiesSegmentDataflow-dataflow_ruby_third_parties_segment_ok_no_datatypes.rb

```diff
@@ -0,0 +1,5 @@
+components: []
+
+
+--
+
```

Comment:
I thought we would have detected the `segment_init` in risks here 🤔 Any thoughts @didroe ?